src/required_resources.h

```diff
@@ -30,6 +30,7 @@ static const auto DIVIDE_BY_ZERO_MSG =
 enum class RuntimeErrorId {
   DIVIDE_BY_ZERO,
   HELPER_ERROR,
+  ERRORF,
```

Comment:
Does this need to be a RuntimeError? I feel like this is just a text stream with annotated info. (See elsewhere my comments for generalized printf.)